qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/definition/QuteDefinitionInWhenSectionTest.java

```diff
@@ -131,6 +131,16 @@ public void definitionInEnumOnOperatorAsEnum() throws Exception {
 				ll("org/acme/MachineStatus.java", r(2, 10, 2, 12), MockQuteProjectRegistry.JAVA_FIELD_RANGE));
 	}
 
+	@Test
+	public void definitionInEnumOnOperatorAsEnumOnly() throws Exception {
+		String template = "{@org.acme.Machine Machine}\r\n" + //
+				"{#when Machine.status}\r\n" + //
+				"  {#is i|n}\r\n" + //
```

Comment:
It should be nice to have 2 new tests:

 * `{#is i|n in}` : it should return nothing.
 * `{#is in i|n}` : it should return a definition.